### PR TITLE
Group

### LINKED ISSUE #1: Implement research group name

### LINKED ISSUE #42: Format base and index for webpage
It may be nice to actually generate the `index.html` and `base.html` pages for the website.
Currently `index.html` is used by the app.
This may also help with #1

### PR DESCRIPTION
**Pull request by CJ-Wright**
Closes #1 and closes #42

### CODE REVIEW

File: regolith/templates/root_index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<h1><a href="#join">Join Us</a></h1>
+Join {{ group['name'] }} by checking out the open positions on our
+<a href="jobs/index.html">jobs page</a> or see if you are interested in any of 
+the following!
+{% include 'jobs_open.html' %}
+
+{% endblock content %}
+{% block footer %}
+Feel free to contact {{ group['name'] }} by emailing {{ group['email'] }}.
+{% endblock footer %}
```
Comment (scopatz) [inline regolith/templates/root_index.html]: newline at end of file please

**Comment by scopatz** [comment]
Thanks for putting this in @CJ-Wright!

**Comment by CJ-Wright** [comment]
I think it errors if `base.html` and `root_index.html` exist. So it would error on most existing websites since the users needed to roll their own.

**Comment by scopatz** [comment]
haha yeah, makes sense

**Comment by CJ-Wright** [comment]
I take that back, this might be ok, since the local template should take precedence over the regolith one.

**Comment by CJ-Wright** [comment]
@scopatz I think this is ready for more review (and seems backwards compatible, I checked it against our website, and it failed due to a bad month on our end but not because of template issues, if you could test it with your website that would be awesome).

**Comment by scopatz** [comment]
This seems to work for me locally, but there is a test failure on CI

**Comment by CJ-Wright** [comment]
Thank you very much!

**Pull request merged**